### ISSUE TITLE
Add instructions on how to get the oauth url without the use of a browser extension

### ISSUE THREAD

**Issue by FeikoJoosten** [arjenbos/ha-postnl#6]
Hi there,

I noticed you stated that you need to use the browser extension to get sent back to home assistant for the oauth callback. While it does make it a lot easier, you don't need to install the extension either. 

If you open the browser console after having logged into post nl, you'll see an error that it couldn't launch an application. It also states the oauth url required. Manually copying that url, replacing the `postnl://login` section with `https://my.home-assistant.io/redirect/oauth` allows you to manually navigate to the newly formatted url. This is then caught by home assistant and the integration successfully installs. Alternatively, you can also enable the networking recorder and catch the oauth url that way. But I don't have example screenshots at hand for that.

Here are some example screenshots.
First login to post nl
![image](https://github.com/arjenbos/ha-postnl/assets/10920052/313e6646-5537-4916-8f68-50976cd7bfa0)

After loggin in you get this blank page
![image](https://github.com/arjenbos/ha-postnl/assets/10920052/145e662a-30ab-4bc6-a37c-4a46b8f26a31)

Open the browser console and find the url
![image](https://github.com/arjenbos/ha-postnl/assets/10920052/02672190-3965-497f-88a5-5157e0415d63)

Copy the url and replace the `postnl://login` section as stated before. Then manually navigate to the new url.

**Comment by FeikoJoosten** [comment]
I would indeed like to see this approach be documented as well. I initially didn't want to add this integration because of the browser extension requirement. After checking the code I saw that all it did was replace a section of an url so I wasn't too bothered by it. But as you're also saying, someone who is not very tech-savvy (or can't read code) won't get that far.

Additionally someone on a Discord I'm in stated they couldn't install the extension either. (This was before you added instruction for installing the extension on different browsers). So I explained how they could use this approach instead.

**Comment by malosaa** [comment]
I did test the browser extension on Brave with 0 issue's
Just unpacked the files, enabled developer mode and imported the code, and works as expected.

Just strange that it won't work for other people, as i did do the same for my other clients.
Like chrome and Mozilla.

**Comment by IIIdefconIII** [comment]
The steps i took
\custom_components\postnl: https://github.com/iMicknl/python-postnl-api
Browser Extention: https://github.com/arjenbos/ha-postnl
Card \www of www\community\postnl-card\: https://github.com/peternijssen/lovelace-postnl-card/tree/0.12.2

Extra info:
https://community.home-assistant.io/t/lovelace-postnl/112433

**Comment by arjenbos** [comment]
> The steps i took \custom_components\postnl: https://github.com/iMicknl/python-postnl-api Browser Extention: https://github.com/arjenbos/ha-postnl Card \www of www\community\postnl-card: https://github.com/peternijssen/lovelace-postnl-card/tree/0.12.2
> 
> Extra info: https://community.home-assistant.io/t/lovelace-postnl/112433

You are aware that you're using a completely different Github repo than this one?

**Comment by arjenbos** [comment]
@IIIdefconIII Are you aware that you're using a completely different Github repo than this one?

**Comment by FeikoJoosten** [comment]
I also don't see how it adds value to the issue itself haha.

**Comment by IIIdefconIII** [comment]
> @IIIdefconIII Are you aware that you're using a completely different Github repo than this one?

yeah i couldnt get it to work yesterday so alot of information got fired on me and those steps actuall got the card and stuff going

**Comment by FeikoJoosten** [comment]
> how do you mean?

You mentioned
> \custom_components\postnl: https://github.com/iMicknl/python-postnl-api
In your initial message. Which is a different repository from this one.

**Comment by IIIdefconIII** [comment]
> > how do you mean?
> 
> You mentioned
> 
> > \custom_components\postnl: https://github.com/iMicknl/python-postnl-api
> 
> In your initial message. Which is a different repository from this one.

i see, why noy merge together?

**Comment by gerard76** [comment]
The path `/redirect/oauth` gives a 404 on my HA install
